recipes/hyperbox-brain/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "hyperbox-brain" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 41ba26a8b285f48ef5f8e76c6f60fdab43b9ed689b7f2c531e7ade4442a83437
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools
+    - numpy
+    - wheel
+    - scikit-learn
+    - pandas
+    - matplotlib-base
+    - joblib
+    - scipy
+    - threadpoolctl
+    - plotly
+  run:
+    - python >=3.6
+    - numpy
+    - scikit-learn
+    - scipy
+    - joblib
+    - threadpoolctl
+    - plotly
+    - pandas
+    - matplotlib-base
+    - setuptools
+
+test:
+  imports:
+    - hbbrain
+    - hbbrain.tests
+    - numpy
+    - pandas
+    - sklearn.datasets
+  requires:
+    - pip
+    - numpy
+    - scikit-learn
+    - pandas
+    - matplotlib-base
+    - pytest
+  commands:
+    - pip check
+
+about:
+  home: https://uts-caslab.github.io/hyperbox-brain/
+  summary: 'A scikit-learn compatible hyperbox-based machine learning library in Python'
+  description: |
+    hyperbox-brain is a Python open source toolbox implementing hyperbox-based machine learning algorithms
+    built on top of scikit-learn and is distributed under the GPL-3.0 license. The project was started in
+    2018 by Prof. Bogdan Gabrys and Dr. Thanh Tung Khuat at the Complex Adaptive Systems Lab - The University 
+    of Technology Sydney. This project is a core module aiming to the formulation of explainable lifelong 
+    learning systems in near future.
+  license: GPL-3.0-only
+  license_family: GPL
+  license_file: LICENSE
+  # The doc_url and dev_url are optional.
```

Comment:
```suggestion
```

Reply:
@carterbox  It has been resolved. Thanks.